manifests/r/ReincubateLtd/iPhoneBackupExtractorSetup/7.3.5.0/ReincubateLtd.iPhoneBackupExtractorSetup.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ReincubateLtd.iPhoneBackupExtractorSetup
+PackageVersion: 7.3.5.0
+Installers:
+- InstallerUrl: https://reincubate.com/res/labs/iphonebe/iphonebackupextractor-latest.exe
+  Architecture: neutral
+  InstallerType: exe
+  InstallerSha256: 57A16B2DD478BB659947E36067D1472197DBBF009B7F2BC681D7F08450510366
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive

```

Comment:
```suggestion
    Silent: /S
    SilentWithProgress: /S
```

Reply:
Despite appearing to be a Windows Installer package, this responds to Nullsoft switches.